src/System.Private.Interop/src/Shared/Interop.Manual.cs

```diff
@@ -58,6 +58,7 @@ internal unsafe partial class COM
         internal static Guid IID_IDispatch = new Guid(0x00020400, 0x0000, 0x0000, 0xc0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x46);
         internal static Guid IID_IManagedActivationFactory = new Guid(0x60D27C8D, 0x5F61, 0x4CCE, 0xB7, 0x51, 0x69, 0x0F, 0xAE, 0x66, 0xAA, 0x53);
         internal static Guid IID_IActivationFactoryInternal = new Guid(0x00000035, 0x0000, 0x0000, 0xC0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x46);
+        internal static Guid IID_ILanguageExceptionStackBackTrace = new Guid(unchecked((int)0xCBE53FB5), unchecked((short)0xF967), 0x4258, 0x8D, 0x34, 0x42, 0xF5, 0xE2, 0x58, 0x33,0xDE);
```

Comment:
I realize all other GUIDs here are in this form, but this form is literally the worst way to show a GUID to a human being (i.e. in source). Are we constructing GUIDs in this manner for performance reasons? If not, can we please use the string form which is easy to verify and consume in almost all cases where humans use a GUID.

Reply:
This is copy&paste of the equivalent C/C++ code ... if you need to initialize Guid in C++, you are probably not going to do it by parsing the strings either.

Reply:
I realize that, but that isn't the human consumable part. Yes, when copying into C++, I agree this is desirable. My normal case is searching for the string or providing documentation. This form is the computer view of the GUID _not_ the human view which is the point of source. Unless perf is a concern here I usually defer to the string format because it is easier to consume. Just my opinion.

Reply:
So put the other format to comment like what is sometimes done in C/C++? e.g. https://github.com/dotnet/coreclr/blob/master/src/inc/cor.h#L271

> Unless perf is a concern here 

This is run early during startup. There are perf and dependency cycles concerns here.

Reply:
Yes. AFAIK, there is a startup perf concern for this.

> , but that isn't the human consumable part
 I can add a commet for these GUID.